.gitignore

```diff
@@ -5,6 +5,7 @@
 
 # TensorFlow Lite model
 *.tflite
+!ESRGAN.tflite
```

Comment:
Please add a scripts folder for downloading the model rather than storing it in the repo. You can use the link https://tfhub.dev/captain-pool/lite-model/esrgan-tf2/1?lite-format=tflite

Reply:
Thanks you, I didn't find the download link before.